README.md

```diff
@@ -55,9 +55,9 @@ It is recommended that you use `virtualenv` (especially for `conda` users).
   * Installing `virtualenv` : ` sudo pip install virtualenv`
   * `virtualenv venv --python=python3`
   * `. venv/bin/activate`
-  * `pip3 install moboff`
+  * `python setup.py install`
 
-### You can also install it system-wide by `pip3 install MobOff` 
+### You can also install it system-wide by `pip install MobOff` 
```

Comment:
I think it's better to remove this line for now, since it's not updated with the current version.